src/test/java/seedu/address/model/client/BirthdayTest.java

```diff
@@ -38,11 +38,15 @@ public void isValidBirthday() {
                 .format(DateTimeFormatter.ofPattern("dd-MM-yyyy")))); //birthday 2 months after current date
         assertFalse(Birthday.isValidBirthday(LocalDate.now()
                 .format(DateTimeFormatter.ofPattern("dd-MM-yyyy")))); // birthday today
+        assertFalse(Birthday.isValidBirthday(LocalDate.now().minusYears(120).minusDays(1)
+                .format(DateTimeFormatter.ofPattern("dd-MM-yyyy")))); // birthday just the day before the earliest limit
 
         // valid birthday
         assertTrue(Birthday.isValidBirthday("01-01-1980")); // birthday in 01-01-1980
         assertTrue(Birthday.isValidBirthday("01-01-1970")); //birthday in 01-01-1979
```

Comment:
Your tests might fail here next time

Reply:
In exactly 70 years time :rofl:

Reply:
our app is gonna be used by my grandchildren